cmd/collector/app/zipkin/http_handler_test.go

```diff
@@ -398,3 +418,9 @@ func postBytes(urlStr string, bytesBody []byte, header *http.Header) (int, strin
 	}
 	return res.StatusCode, string(body), nil
 }
+
+func randBytesOfLen(n int) []byte {
+	b := make([]byte, n)
+	rand.Read(b)
```

Comment:
I think we should add a `require.NoError` here.

Reply:
+1